client/Sheepshead/SharedSheep/Round/Round.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using SharedSheep.Player;
+using SharedSheep.Trick;
+
+namespace SharedSheep.Round
+{
+    class Round : IRound
+    {
+        public ITrick Trick { get; set; }
+        public int RoundNumber { get; set; }
+        public IPlayer RoundStarter { get; set; }
+
+        public Round() { }
+
+        public Round(ITrick trick, int roundNum, IPlayer roundStarter)
```

Comment:
The empty constructor I don't think is useful. And I think this constructor shouldn't have trick. When you create a round, that round will create an empty trick and fill it as the round progresses.